### ISSUE TITLE
enable to go back to previous steps of registration workflow

### ISSUE THREAD

**Issue by sdebodt** [ugent-library/biblio-backend#320]
**Describe the improvement**
When adding a publication, I want to be able to return to a previous step. Now the DOI disappears and needs to be entered again. As such multiple draft records are created.

Also check behavior if back button of browser is used.

**Comment by nicolasfranck** [comment]
The use of the browser "back" button isn't advised, as it loads a web page from cache and prefills input data from its history. Any possible related events (like a dialog) will reappear. That is outside of our control. It is sort of "go back in history". Users indeed often do that, but are surprised to see that the page is sometimes different when they actually reload it.

Going back to a previous step in a workflow is not easy: 

* every step is dependent on the previous one. Change one step, and the other step possibly doesn't exist anymore (e.g. switch from "one publication" to "multiple publications"). Compare it to a route in steps.
* it can lead to unmanageable expectations. e.g. return to the step where you did not yet pushed the button "publish". In which case you would expect the publications not to be published, right? That is why we lock you in a certain state of wizard, as otherwise we have to record every previous state of every publication also. And what if you return the last step? Hard to follow ;-)

Why not add a "cancel"/"reset" button that removes all current data? For that is basically the effect of resetting the DOI.

**Comment by sdebodt** [comment]
with the new "start" step implemented, no further action needed for this issue.
